src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -44,3 +44,4 @@
 [assembly: InternalsVisibleTo("Tester.SQLUtils")]
 [assembly: InternalsVisibleTo("Tester.ZooKeeperUtils")]
 [assembly: InternalsVisibleTo("AWSUtils.Tests")]
+[assembly: InternalsVisibleTo("ServiceBus.Tests")]
```

Comment:
What happens if InitGlobalBufferPool is not called?
If we need test setup of the buffer pool, I suggest we add it to OrleansTestingHost, so it's available to all test harnesses without the need to add internal access.